src/components/DataMessenger/DataMessenger.js

```diff
@@ -207,6 +210,26 @@ export default class DataMessenger extends React.Component {
       .catch((error) => {
         console.error(error)
       })
+
+    if (this.props.enableQueryQuickStartTopics) {
+      fetchTopics(getAuthentication(this.props.authentication))
+        .then((response) => {
+          this.setState({
+            topics: _get(response, 'data.data.topics'),
+          })
+          if (_get(this.state, 'topics.length') > 0) {
+            const topicsMessageContent = this.createTopicsMessage()
+            if (topicsMessageContent) {
+              this.state.messages.push(
+                this.createIntroMessage({ content: topicsMessageContent })
+              )
```

Comment:
We should never mutate the state directly (same goes for props. React is designed to be in full control of state and props). Whenever we want to change the state we should always use `setState` 

so something like
`const topicsMessage = this.createIntroMessage({ content: topicsMessageContent })
this.setState({
   messages: [
      ...this.state.messages,
      topicsMessage
   ]
})
`